packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -219,6 +219,10 @@ export const styles = css`
         place-items: center;
     }
 
```

Comment:
I added @mollykreis as a reviewer since she's been the main person in the previous discussions on the linked issue.

From my reading of her comment, I understood the suggestion was to make this change in a more common location like our button styles, not just in the rich text editor styles.

Reply:
This is more of like a workaround fix for the `nimble-rich-text-editor` as the component is already integrated into the SLE, We thought of resolving some of the known issues if we have any workarounds in hand. Solving this workaround in the common location for buttons might not be the proper fix as the button slot contents can be anything, but we are targeted for only icons. Here in `rich-text-editor`, the buttons will only be used with icons placed at `slot="start"` as of now and once it is fixed in the actual component, we will remove this workaround. 

I have added a placeholder comment linking the issue and once it is fixed, we will remove this workaround.

Reply:
My opinion is that the fix in this PR is too narrowly focused, even for a workaround. I don't have a problem adding `pointer-events: none;`, but as @jattasNI mentioned, I think we should add that style in a more common location, such as the shared styles for all buttons (https://github.com/ni/nimble/blob/main/packages/nimble-components/src/patterns/button/styles.ts).

Reply:
Thanks for adding a code comment linking to the issue. If we go forward with this approach, could you also update the issue to mention that this workaround should be removed when the issue is fixed?

As for whether we *should* go forward with the workaround.... It's technically true that a client can slot anything into a button. But in our documentation we only describe adding icon content. And in a few spots in the code we apply icon-specific styling (e.g. setting icon colors). So for now it is our policy that the only supported content in that slot is an icon and I would be comfortable with adding more icon-specific code to the buttons. (Other reviewers, feel free to weigh in).

If you have the bandwidth to make the fix in the button code, that would be my preference (maybe in a separate PR so we can compare them). If not, I agree that fixing this case is better than doing nothing so I will approve it.

Reply:
I will wait for other reviewer's thoughts on this. If everyone is fine with this approach, I will update the issue to remove this workaround when the actual fix is implemented.

Reply:
@rajsite and @mollykreis and I agreed at our team meeting today that it makes sense to proceed with this PR as-is since we don't have a super clear plan for what the actual fix will look like.

Reply:
I have updated the description of this issue https://github.com/ni/nimble/issues/1422#issue-1844688796 to remove this workaround when the actual fix is implemented.